src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,18 @@
 import React from 'react';
+import './UserInfo.scss';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
+import { User } from '../../types/user';
+
+type UserType = {
+  user: User;
+};
+
+export const UserInfo: React.FC<UserType> = ({ user }) => (
+  <p>
+    {' Posted by  '}
+
+    <a className="UserInfo" href={user.email}>
```

Comment:
```suggestion
    <a className="UserInfo" href={`mailto:${user.email}`}>
```
All emails should clickable links. Fix all cases
![image](https://github.com/mate-academy/react_static-list-of-posts/assets/96409563/3b44f4e3-2d6a-4935-be4c-6cd70ed55c97)